src/ai/backend/manager/models/vfolder.py

```diff
@@ -1210,6 +1211,178 @@ async def batch_load_by_user(
                 lambda row: row["user"],
             )
 
+    @classmethod
+    async def load_count_invited(
+        cls,
+        graph_ctx: GraphQueryContext,
+        *,
+        domain_name: str = None,
+        group_id: uuid.UUID = None,
+        user_id: uuid.UUID = None,
+        filter: str = None,
+    ) -> int:
+        from .user import users
+
+        j = vfolders.join(
+            vfolder_permissions,
+            vfolders.c.id == vfolder_permissions.c.vfolder,
+        ).join(
+            users,
+            vfolders.c.user == users.c.uuid,
+            isouter=True,
+        )
```

Comment:
```suggestion
        ).join(
            users,
            vfolder_permissions.c.user == users.c.uuid,
        )
```
This would be better!